src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.tsx

```diff
@@ -26,6 +26,7 @@ import { styles } from '../parameters';
 import Alert from '@mui/material/Alert';
 
 const VoltageLimitsParameters = () => {
+    const DndTableTyped = DndTable as React.ComponentType<any>;
```

Comment:
This is strange no ?

Reply:
move outside of component + added a comment to fix later